homeassistant/components/aladdin_connect/translations/en.json

```diff
@@ -0,0 +1,15 @@
+{
+    "config": {
```

Comment:
Run `script.translations develop` to generate a new en.json file from your strings.json

Reply:
I did run this earlier: python3 -m script.translations develop --integration aladdin_connect
Is that right?  Looks like it modified en.json this time too.